docs/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -282,6 +282,29 @@ The corresponding environment variable in the
 [ksqlDB Server image](https://hub.docker.com/r/confluentinc/ksqldb-server/)
 is `KSQL_KSQL_FUNCTIONS_SUBSTRING_LEGACY_ARGS`.
 
+### ksql.persistence.default.format.key
+
+Sets the default value for the `KEY_FORMAT` property if one is
+not supplied explicitly in [CREATE TABLE](../../../developer-guide/ksqldb-reference/create-table.md)
+or [CREATE STREAM](../../../developer-guide/ksqldb-reference/create-stream.md)
+statements.
+
+The default value for this configuration is `KAFKA`.
```

Comment:
I'll add this for consistency but it's not actually possible for there to be no default key format right now, since the default server config contains a default and there's no way to override that with "no default" currently.